CLI/PkModeling.xml

```diff
@@ -126,7 +126,7 @@
       <description><![CDATA[Input 4D Image (txyz)]]></description>
     </image>
 
-    <image type="label">
```

Comment:
yep, I had trouble loading the already existing masks in slicer, because they were not declared label images. Later I found out that 1. the command line doesn't care (this could be a bug!?), 2. it's just the name of the file that needs to have "label" added to the name to be recognized by slicer ... and then i forgot to undo my change to the xml ...